### ISSUE TITLE
UC 7: As a developer, I want to be able to get the context of my application

### ISSUE THREAD

**Issue by jorgemoralespou** [redhat-developer/ocdev#152]
As a developer using the command line tool, I want to be able to get the context of the application I’m in.

```
$ ocd app
You are currently working on the XYZ application

$ ocd app --short
XYZ
```

__*NOTE:*__ The context can be inserted in a PS1 if the developer wants to customize it

**Comment by slemeur** [comment]
Story needs to be completed by fixing #190

**Comment by kadel** [comment]
this has been already  implemented and #190 is fixed. Closing this
